src/accommodation/accommodation.controller.ts

```diff
@@ -111,43 +112,42 @@ export class AccommodationController {
   @ApiConsumes('multipart/form-data')
   @ApiBody({
     schema: {
-      type: 'object',
-      properties: {
-        file: {
-          type: 'string',
-          format: 'binary',
-          description: 'Image file (only .jpg, .jpeg, .png allowed), size < 10MB!',
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          filename: { type: 'string' },
+          base64Image: { type: 'string' },
         },
       },
+      description: 'Images file (only .jpg, .jpeg, .png allowed), size < 10MB!',
     },
   })
   @ApiBearerAuth()
   @UseGuards(UserGuard)
   @LangQuery()
   @Post('/:id/file')
-  @UseInterceptors(FileInterceptor('file'))
+  @UseInterceptors(FilesInterceptor('images', 10))
   async updateAccommodationAddFile(
-    @UploadedFile(
+    @UploadedFiles(
       new ParseFilePipe({
         validators: [
-          new MaxFileSizeValidator({ maxSize: 10 * 1024 * 1024 }), // 10MB
+          new MaxFileSizeValidator({ maxSize: 10 * 1024 * 1024 }),
           new FileTypeValidator({ fileType: '.(png|jpeg|jpg)' }),
```

Comment:
looks like this is place to create constant for file types. You use the sample here and above in description